pkg/kubelet/volumes.go

```diff
@@ -285,10 +285,6 @@ func (kl *Kubelet) newVolumeAttacherFromPlugins(spec *volume.Spec, pod *api.Pod,
 	if err != nil {
 		return nil, fmt.Errorf("can't use volume plugins for %s: %v", spec.Name(), err)
 	}
-	if plugin == nil {
```

Comment:
Correct for the Attachable bit.  This is for the volume plugin itself.

I think returning `nil, nil` is an error at this point.  No plugin was found to satisfy the volume spec.